packages/pds-ember/app/styles/pds/components/icon/__config.scss

```diff
@@ -3,22 +3,32 @@ $_module: "PDS.Components.Icon";
 
 $color: currentColor;
 
-@mixin appearance {
-  /* [debug] #{$_module}@appearance */
+@mixin layout {
+  /* [debug] #{$_module}@layout */
   display: inline-block;
   flex-shrink: 0;
   font-size: inherit;
   height: 1em;
-  line-height: 1;
+  line-height: inherit;
```

Comment:
This is important so that the svg wrapper aligns properly with surrounding text content.

Reply:
Here's what it looks like without the change (vertical alignment of the icon is broken)...
![CleanShot 2020-10-23 at 16 34 16](https://user-images.githubusercontent.com/545605/97056031-9f0c9880-154d-11eb-8765-0672c39c5aef.png)

... and with the change (vertical alignment corrected)
![CleanShot 2020-10-23 at 16 35 07](https://user-images.githubusercontent.com/545605/97056072-b8154980-154d-11eb-8570-e3a270d4f552.png)